### DISCUSSION TITLE
smart_filter using filepath / available FilteringFields

### DISCUSSION THREAD

**Discussion by benfishbus** [meisnate12/Plex-Meta-Manager#595] [Q&A] [answered]
I only see where PMM's filter function can use filepath. Is there any way to use filepath with Plex smart search/filter? It's not listed in Plex advanced filter criteria, but somebody on reddit claimed that Folder Name is in the API. Are all FilteringFields available to PMM plex/smart builder, and if so could PMM optionally report out the list of available fields for a specified library?

**Comment by meisnate12** [comment, marked as answer]
the folder name is in the API thats how i can access and use it with filters but just because the data is there doesnt mean it will work with a `plex_search`/`smart_filter`. Plex would need to program into those searches the ability to look at filepath nothing i do will change how that works. Im not going to have PMM print out a list of attributes because well thats why i made the wiki see [here](https://github.com/meisnate12/Plex-Meta-Manager/wiki/Plex-Builders#plex-search) for all available attributes that can be used with `plex_search`/`smart_filter`. It has every one available in the UI.
